mod/gc_onboard/views/default/profile-steps/intro.php

```diff
@@ -5,26 +5,29 @@
  * First part loaded of profile module. Presents the user with message as to what this module will do for them.
  */
 ?>
-<div class="col-md-8 clearfix" id="step">
-    <h1>
-        <?php echo elgg_echo('onboard:profile:intro:title'); ?>
-    </h1>
+<h1 class="wb-inv"><?php echo elgg_echo('profilemodule:title'); ?></h1>
+<div tabindex="-1" class="col-md-8 clearfix" id="step" aria-live="assertive">
+    <h2>
+        <?php echo elgg_echo('onboard:profile:intro:title'); echo elgg_view('page/elements/step_counter', array('current_step'=>1, 'total_steps'=>6, 'class' => 'mrgn-tp-md wb-inv'));?>
+    </h2>
     <p>
         <?php echo elgg_echo('onboard:profile:intro:description'); ?>
     </p>
 
-    <a class="mrgn-bttm-md mrgn-tp-lg btn btn-primary btn-lg pull-right" href="#" id="start">
+    <button type="button" class="mrgn-bttm-md mrgn-tp-lg btn btn-primary btn-lg pull-right" id="start">
         <?php echo elgg_echo('onboard:profile:intro:start'); ?>
-    </a>
+    </button>
 </div>
 
 <div class="col-md-4">
     <div class="panel panel-custom">
         <div class="panel-heading"> <h2 class="panel-title"><?php echo elgg_echo('onboard:steps'); ?></h2></div>
-        <div class="panel-body" id="step-progress">
+        <div class="panel-body" id="step-progress" aria-hidden="true">
```

Comment:
Is there a reason to hide this from AT? It looks like the view itself is adding it https://github.com/gctools-outilsgc/gcconnex/blob/60cd3b98b17d9257870f72e52169ff710137ee9e/mod/gc_onboard/views/default/page/elements/step_counter.php#L29.
Might consider https://css-tricks.com/html5-progress-element/ instead